freqtrade/resolvers/iresolver.py

```diff
@@ -83,7 +87,7 @@ def _search_object(cls, directory: Path, object_name: str
                 continue
             module_path = entry.resolve()
 
-            obj = next(cls._get_valid_object(module_path, object_name), None)
+            obj = next(cls._get_valid_object(module_path, object_name), None)  # noqa
```

Comment:
I don't understand the reason for this `# noqa`

I've tried to remove it and it doesn't seem to cause errors in either mypy nor flake8?

Reply:
It's left after https://github.com/freqtrade/freqtrade/pull/2923/commits/c92e1d97d65b796a061d19081f701cac6aa59ce3, forgotten to remove it...